### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by Guy-deMessimy**

### CODE REVIEW

**Review by AmjadLille — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by AmjadLille — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -57,6 +57,5 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "prettier": "^2.2.1"
-  },
-  "proxy": "http://localhost:5000"
+  }
```
Comment (LoicBrassart) [inline package.json:60]: Untested, will break ! :warning:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Breakings included, fix it (and **read** before you assign me !)

**Review by LoicBrassart — APPROVED** [review]
Nevermind, I'm tired!

**Pull request merged**